content/docs/studio/user-guide/projects-and-experiments/what-is-a-project.md

```diff
@@ -19,6 +19,9 @@ hyperparameters and models. These values are then presented in an experiment
 table with each experiment (Git commit) in a row and the corresponding values
 for the data, metrics, hyperparameters and models in columns.
 
+Live metrics that you [send to Iterative Studio][live-metrics-and-plots] using
```

Comment:
Plots?

Reply:
This is about the project table. The table includes the metrics but not the plots. So I think it is ok.